packages/backend-defaults/config.d.ts

```diff
@@ -609,6 +617,68 @@ export interface Config {
       headers?: { [name: string]: string };
     };
 
+    /**
+     * Rate limiting options. Defining this as `true` will enable rate limiting with default values.
+     */
+    rateLimit?:
+      | true
+      | {
+          store?:
+            | {
+                client: 'redis';
```

Comment:
Thinking that `type` might be a better name for this key? Similar to for example `backend.auth.pluginKeyStore`

Reply:
Sure. Also was wondering if we should have the automatic config for the store in case it's not defined; first check the `cache.store` for `redis` connection, and then use `postgresql` from `database.client` and lastly use memory store. But this is something to improve later.